### PR TITLE
update tags for A4Education

### PR DESCRIPTION
**Pull request by vinnieangel**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by ecfan — CHANGES_REQUESTED** [review]
Hi @vinnieangel, thanks for your contribution. We need to keep the Consumption label to clarify that this template doesn't create a Standard logic app.

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "icon": "logic",  
-  "itemDisplayName": "Create a Consumption logic app workflow with an ARM template",
-  "description": "This template creates an empty Consumption logic app for defining a workflow.",
+  "itemDisplayName": "Create a logic app by using a template",
+  "description": "This template creates an empty logic app that can be used to define workflows.",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:5]: A Consumption logic app lets you create only one workflow versus a Standard logic app allows multiple workflows.

```suggestion
  "description": "This template creates an empty Consumption logic app to define a workflow.",
```

File: quickstarts/microsoft.logic/logic-app-create/metadata.json [hunk 2]
```diff
@@ -13,6 +13,11 @@
     "resource": "Logic Apps, MySQL",
     "os": "Linux, Logic App",
     "level": "beginner",
-    "cost": "$"
+    "cost": "$",
+    "scenario": "Azure Logic App",
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/metadata.json:17]: Fix the product name:

```suggestion
    "scenario": "Azure Logic Apps",
```

**Review by ecfan — CHANGES_REQUESTED** [review]
Please accept the proposed edits, thanks!

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.logic/logic-app-create/metadata.json (hunk 1 above)
Comment (ecfan) [reply quickstarts/microsoft.logic/logic-app-create/metadata.json:5]: Missed one here.

**Pull request merged**